src/main/resources/lib/guillotine/index.js

```diff
@@ -106,12 +137,16 @@ function invalidate(siteId, branch) {
 }
 
 function getSchema(siteId, branch, schemaOptions) {
-    let schemaId = `${siteId}/${branch}`;
+    let schemaId = siteId === null ? `__PROJECT_SCHEMA_ID__` : `${siteId}/${branch}`;
```

Comment:
The seem to be a problem, that schemaId does not include options.

Reply:
No problem here, we can discuss about it